### DISCUSSION TITLE
Meta_Parser在移动端能直接使用吗？ 该怎么编译？

### DISCUSSION THREAD

**Discussion by wanglelewang** [BoomingTech/Piccolo#311] [Q&A] [answered]
**Describe the bug | Bug描述**

**Steps to reproduce | 如何复现**
1. 
2. 
3. 
4. 
5. 

**Expected behavior | 预期行为**
    
**Actual behavior | 实际行为**
A clear and concise description of what actually happened.

**Screenshots/Logs | 截图/日志**
If applicable, add screenshots and/or a video to help explain your problem.

**Desktop | 桌面环境**
 - OS: [e.g. Windows, macOS, Linux]
 - Version [e.g. Win10, Bug Sur, Ubuntu20]
 - CPU [e.g. Intel I9-9900k , Ryzen 5900x, ]
 - GPU [AMD 6800 XT, NVidia RTX 3090]
 - Memory [e.g. 16GB]
 - Compiler [e.g. MSVC2019, Clang13, G++7.0]

**Additional context | 补充信息**

**Comment by wanglelewang** [comment]
hello？

**Comment by Ol6rin** [comment, marked as answer]
你好，目前PiccoloEngine还不支持移动端，暂时不在开发计划内。
